althea_types/src/contact_info.rs

```diff
@@ -95,37 +117,53 @@ pub struct ContactStorage {
     email: Option<EmailAddress>,
     invalid_number: Option<String>,
     invalid_email: Option<String>,
+    sequence_number: u32,
```

Comment:
👍🏻  

we'll need to add serde default as a tag to this so that it starts at zero.

Serde default means that if serde does not find the field it populates it with that value, this will 'add it in' to all the existing routers configs. For ContactType we'll still use an option so ops tools can identify the older routers that are not sending a sequence number.